SafeguardSessions/Test/Unit/Search/TestOpenSnapshot.query.pq

```diff
@@ -0,0 +1,75 @@
+section TestOpenSnapshot;
+
+OpenSnapshot = Extension.ImportModule("OpenSnapshot.pqm");
+
+OpenSnapshot.OpenSnapshot = OpenSnapshot[OpenSnapshot];
+
+TestOpenSnapshotIsSuccessful = () =>
+    let
+        fakeSnapshotResponse = FakeRawResponse([
+            body = [snapshot = "dummy_snapshot_id"]
+        ]),
+        snapshot = OpenSnapshot.OpenSnapshot(
+            "dummy_url", "dummy_session_id", [
+                OpenSnapshotResponse = fakeSnapshotResponse
+            ]
+        )
+    in
+        Fact("Snapshot is returned correctly", "dummy_snapshot_id", snapshot);
+
+TestOpenSnapshotRaisesError = () =>
+    let
+        AsserErrorIsRaised = (
```

Comment:
It's working, but for clarity we shouldn't name it AssertErrorIsRaised since UnitTestFrameWork already defines AssertErrorIsRaised.